packages/code-editor/README.md

```diff
@@ -16,25 +17,38 @@ npm install @looker/code-editor
+| code        | The code to be displayed and highlighted.                                                                                                                                                                                                                                  | _required_ |
+| pattern     | The search pattern to be highlighted in the input code.                                                                                                                                                                                                                    | `''`       |
+| transparent | A flag for disabling the backgroundColor added by default.                                                                                                                                                                                                                 | `false`    |
+| inline      | When true, inline `<code />` is used. `<pre />` is used by default.                                                                                                                                                                                                        | `false`    |
+| lineNumbers | A flag for enabling line numbers.                                                                                                                                                                                                                                          | `false`    |
+
+### `<CodeCopy />`
+
+This component wraps the `<CodeDisplay />` component and adds a "copy to Clipboard" button
+
+| Prop | Description | Default |
 | ----------- | ----------- | ----------- |
-| language      | This prop configures the syntax highlighting language and accepts any `react-prism-renderer` [supported language](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js), as well as `ruby`, `kotlin`, `swift`, and `csharp`       | `json` |
-| code   | The code to be displayed and highlighted.        | _required_ |
-| pattern   | The search pattern to be highlighted in the input code.         | `''` |
-| transparent   | A flag for disabling the backgroundColor added by default.        | `false` |
-| inline   | When true, inline `<code />` is used. `<pre />` is used by default.        | `false` |
-| lineNumbers   | A flag for enabling line numbers.        | `false` |
+| ... | All `<CodeDisplay />` props are accepted | |
+| caption | An override for the Copy button text, which is "Copy" | _optional_ |
 
 ### `<CodeEditor />`
-This component extends the `<CodeDisplay />` component and adds a hidden `<textarea />` that allows for code editting.
-| Prop      | Description | Default |
+
+This component extends the `<CodeDisplay />` component and adds a hidden `<textarea />` that allows for code editing.
```

Comment:
extends or wraps in this scenario? props extends, component wraps :D . Went back and looked at CodeCopy and its the same thing. My gut feeling is that technically it should be wraps but I can live with extends because of the props.

Gonna approve anyway

Reply:
Not gonna change this one right now. We can update when we figure out how to support the copy clipboard button better than `CodeCopy` does

Reply:
Just updated this because I needed to revert `clipboard` in types.ts